docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,79 @@
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Code side-by-side.  It provides several options to organize projects to help developers achieve this goal.
+
+## General Guidance
```

Comment:
Is this a process or options? The numbered list kind of gives an idea that it is a process to follow. If not, I'd suggest to change to bullet points.
Also, maybe an introductory sentence here would give more context like:
"When organizing your projects, there are different strategies you can use:"